src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -427,21 +530,24 @@ def calibrate(
         `caf.distribute.furness.doubly_constrained_furness()`
         `scipy.optimize.least_squares()`
         """
-        for dist in self.dists:
+        for dist in distributions:
             self.cost_function.validate_params(dist.function_params)
         self._validate_running_log(running_log_path)
         self._initialise_internal_params()
         return self._calibrate(  # type: ignore
+            distributions,
+            running_log_path,
             *args,
-            running_log_path=running_log_path,
             **kwargs,
         )
 
     def _jacobian_function(
         self,
         init_params: list[float],
         cost_distributions: list[MultiCostDistribution],
+        furness_tol: int,
         diff_step: float,
+        furness_jac: bool,
         running_log_path,
         params_len,
```

Comment:
Type annotations for these, I think path and int?
```suggestion
        running_log_path: Path,
        params_len: int,
```